modules/rest-util/test/blaze/middleware/fhir/output_test.clj

```diff
@@ -205,5 +220,26 @@
       [:headers "Content-Type"] := "application/fhir+xml;charset=utf-8"
       [:body parse-xml :issue 0 :diagnostics] := "Invalid white space character (0x1e) in text to output (in xml 1.1, could output as a character entity)")))
 
+(deftest binary-test
+  (testing "with data and without content type"
+    (given (call (binary-resource-handler-200 {:content-type nil :data #fhir/base64Binary"MTA1NjE0Cg=="}) {:headers {"accept" "text/plain"}})
+      :status := 200
+      [:headers "Content-Type"] := "application/octet-stream"
+      [:body bs/from-byte-array] := #blaze/byte-string"3130353631340A"))
+  (testing "with data and with content type"
+    (given (call (binary-resource-handler-200 {:content-type "text/plain" :data #fhir/base64Binary"MTA1NjE0Cg=="}) {:headers {"accept" "text/plain"}})
+      :status := 200
+      [:headers "Content-Type"] := "text/plain"
+      [:body bs/from-byte-array] := #blaze/byte-string"3130353631340A"))
+  (testing "without data and with content type"
+    (given (call (binary-resource-handler-200 {:content-type "text/plain" :data nil}) {:headers {"accept" "text/plain"}})
+      :status := 200
+      [:headers "Content-Type"] := "text/plain"
+      [:body := nil]))
```

Comment:
`:= nil` has to be outside of the path vector. in this case you don't need the vector at all like with status

Reply:
`:body := nil`, then?